tox.ini

```diff
@@ -13,6 +10,14 @@ commands =
    python -V
    nosetests []
 
+[testenv:docs]
+commands =
+   python setup.py build_sphinx
+
+[testenv:update-pep8]
+commands =
+   {toxinidir}/scripts/update-pep8.sh
```

Comment:
On thing that is not clean for me is, do we fail if update-pep8.sh change any file on travis? If we don't we should add verify-pep8.sh the same way we have verify in k8s and call that one.